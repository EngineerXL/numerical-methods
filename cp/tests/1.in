6351874531919111042693229
3441517150878825885885978
