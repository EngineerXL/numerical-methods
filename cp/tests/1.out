21860085141830034828733524396926129922815926642962
